Jenkinsfile.release

```diff
@@ -133,9 +138,11 @@ spec:
             // Maven required git push permission on https://github.com/jenkinsci/jenkins
             sshagent(['release-key']) {
               // Maven Release requires gpg key with password password and a certificate key with password
-              sh 'scripts/buildJenkins.bash --configureGPG'
-              sh 'scripts/buildJenkins.bash --configureGit'
-              sh 'scripts/buildJenkins.bash --prepareRelease'
+              sh '''
+                scripts/buildJenkins.bash --configureGPG
+                scripts/buildJenkins.bash --configureGit
+                scripts/buildJenkins.bash --prepareRelease
+              '''
               script {
```

Comment:
BTW you could get rid of the `script` block by externalizing this processing: the script doing the `git push` could just load `scm.tag` from `release.properties` itself.

Reply:
Do you mean by letting maven release to handle it from pom.xml?
```
  <scm>
    <connection>scm:git:git://github.com/jenkinsci/jenkins.git</connection>
    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/jenkins.git</developerConnection>
    <url>https://github.com/jenkinsci/jenkins</url>
    <tag>${scmTag}</tag>
  </scm>
```

Reply:
I am not sure to understand your point here

Reply:
I mean that if you deleted this `script` block, the corresponding shell script fragment

```sh
git push origin $RELEASE_SCM_TAG
```

could do the equivalent computation on its own by parsing `release.properties`.